Products/ZenHub/server/config.py

```diff
@@ -59,6 +61,34 @@ def xmlrpcport(self):
         return self.__config.xmlrpcport
 
 
+class ZenHubServiceCallConfig(object):
+    """ load and store configuration from file """
+    
+    def __init__(self, config):
+
+        self.__config = {}
+
+        try:
+            with open(config, 'r') as f:
+                self.__config = json.load(f)
+        except Exception as e:
+            import logging
+            log = logging.getLogger("zen.%s" %  __name__)
```

Comment:
Please name this logger "zen.zenhub.server.config" to stay consistent with the other loggers defined in zenhub, e.g.
```
    log = logging.getLogger("zen.zenhub.server.config")
```
The `__name__` variable will return "Products.ZenHub.server.config" which is not the desired name.

Reply:
done